packages/core/src/messagingProtocols/proteus/ProteusService/DecryptionErrorGenerator/DecryptionErrorGenerator.ts

```diff
@@ -18,46 +18,60 @@
  */
 
 import {QualifiedId} from '@wireapp/api-client/lib/user';
-import logdown from 'logdown';
 
 import {DecryptionError} from '../../../../errors/DecryptionError';
 
 type ErrorWithCode = Error & {code?: number};
 const hasErrorCode = (error: any): ErrorWithCode => error && error.code;
```

Comment:
i know this isnt something you changed but i find this function and its usage sort of confusing. in my opinion it should either be a boolean or it shouldn't be used in ternaries like on line 45. wdyt?